encord/client_metadata_schema.py

```diff
@@ -0,0 +1,53 @@
+import json
+import uuid
+from typing import Any, Dict, Optional
+from uuid import UUID
+from datetime import datetime
+import encord.orm.client_metadata_schema as orm
+from encord.http.v2.api_client import ApiClient
+
+
+class ClientMetadataSchema:
```

Comment:
Probably a matter of taste, but do you think we need this object at this point? 
Would just having "get_client_metadata_schema" and "set_client_metadata_schema" on the user client level, that would work with ClientMetadataSchema be more straightforward? 

I'd say that having such an object makes sense if we expect a user to make a long repeated interaction with the entity (e.g. like with Project). In this case it seems like save/load anyways?